src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryApplication.groovy

```diff
@@ -17,15 +21,37 @@ class DirectoryApplication extends Application<DirectoryApplicationConfiguration
     public void run(DirectoryApplicationConfiguration configuration, Environment environment) {
         this.setup(configuration, environment)
 
+        def ldapConnectionPool = configureLdapPool(configuration.ldapConfiguration)
```

Comment:
Why not use a `PooledConnectionFactory` instead of `def`? Your `configureLdapPool` method could also return this type too. I'm good either way, just curious

Reply:
Oh good catch on the method return type. That's fixed.

As for the def I left it that way because having the full type in conflicts with the codenarc line length rules.

Reply:
Gotcha. I guess I would prefer using a more specific type whenever possible. You could split it over multiple lines, codenarc wouldn't complain about this:
```
PooledConnectionFactory ldapConnectionPool = configureLdapPool(
        configuration.ldapConfiguration)
```
I'm good either way though, feel free to change it if you want to.